ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -3693,3 +3693,35 @@ def test_paddle_tensor_zero_(
         frontend=frontend,
         on_device=on_device,
     )
+
+# quantile
+@handle_frontend_test(
```

Comment:
This was never imported...the other tests are using a different decorator in case you did not realise!

Reply:
I have done that what you mentioned!